opennms-doc/guide-admin/src/asciidoc/text/application-perspective-monitoring/application-perspective-monitoring.adoc

```diff
@@ -2,36 +2,36 @@
 These remote locations are called perspective locations in this context.
-In order to configure remote polling an user has to define an application by associating a set of services and a set of perspectives from which these services should be monitored.
+In order to configure _Application Perspective Monitoring_ an user has to define an application by associating a set of services and a set of perspectives from which these services should be monitored.
 
-.Remote Poller operation
-image::remote-poller/rp-operation.png[]
+.Application Perspective Monitoring operation
+image::application-perspective-monitoring/apm-operation.png[]
 
-The _Remote Poller_ utilizes the minion infrastructure to monitor a service's availability from these different perspectives.
+_Application Perspective Monitoring_ utilizes the minion infrastructure to monitor a service's availability from these different perspectives.
 The service monitor configuration is looked up in the `poller-configuration.xml`.
 
-NOTE: The `remote-only` flag in the package definition can be used to define packages that will be used *only* for remote polling.
+NOTE: The `remote-only` flag in the package definition can be used to define packages that will be used *only* for _Application Perspective Monitoring_.
 
 When a service is not responsive an outage will be generated with the corresponding perspective.
-In order to configure remote polling first an application has to be created.
+In order to configure _Application Perspective Monitoring_ first an application has to be created.
 
 .Create a new application
-image::remote-poller/create-app.png[]
+image::application-perspective-monitoring/create-app.png[]
 
 Next step is to configure the application.
 In the upper section services that should be monitored from remote perspective locations can be associated with this application.
 In the lower section the perspective locations from which the given services should be monitored can be defined.
 
 .Configuration of an application
-image::remote-poller/configure-app.png[]
+image::application-perspective-monitoring/configure-app.png[]
 
 After configuring the application, the minions at the remote perspective locations start to monitor the services associated with this application.
-The next screenshot show an HTTP outage noticed from all perspective locations and the _{opennms-product-name}_ instance itself.
+The next screenshot show an HTTP outage noticed from all perspective locations and the _{opennms-product-name}_ poller daemon itself.
 
 .Remote outages
-image::remote-poller/remote-outages.png[]
+image::application-perspective-monitoring/remote-outages.png[]
 
 The column _Perspective_ shows the remote location from which a minion has detected this outage.
 If no _Perspective_ is shown, the outage was detected by the normal process: the _{opennms-product-name}_ instance detected it in the _Default_ location or a minion detected it in the corresponding node's location.
```

Comment:
```suggestion
An empty *Perspective* column indicates that the normal process detected the outage: either the _{opennms-product-name}_ instance detected it in the default location or a Minion detected it in the corresponding node's location.
```
Please verify that saying the column is empty is correct. Would it be blank, or a null value?

Reply:
We don't know it yet. Maybe we will use an icon or term when no perspective is given. I'll change this accordingly when we have a final solution.